src/components/PlanCard/PlanCard.styled.tsx

```diff
@@ -11,9 +11,9 @@ export const CardTopNav = styled.section`
   justify-content: space-between;
   align-items: center;
   background: ${({ theme }) => theme.color.white};
-  color: ${({ theme }) => theme.color.primary};
+  color: ${({ theme }) => theme.color.black};
   font-size: 1.1rem;
-  font-weight: 600;
+  font-weight: 700;
```

Comment:
I also kept this as 600 in my PR #138 , iirc 700 will make the text very bold, a bit different from figma

Reply:
I changed this back to 600!